.github/workflows/coverage.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+name: Coverage CI
+
+on:
+  [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: [ubuntu-latest, macos-latest, ubuntu-16.04]
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
```

Comment:
Again, I believe 9 tests is too much for the library that doesn't have any platform-specific code.
We can test only on latest Node for macOs an test on windows instead of old Ubuntu:
```yml
    strategy:
      matrix:
        os: [ubuntu-latest, windows-latest, macOS-latest]
        node: [14.x, 12.x, 10.x]
        exclude:
          # On Windows, run tests with only the LTS environments.
          - os: windows-latest
            node: 10.x
          - os: windows-latest
            node: 14.x
          # On macOS, run tests with only the LTS environments.
          - os: macOS-latest
            node: 10.x
          - os: macOS-latest
            node: 14.x

    runs-on: ${{ matrix.os }}
```